src/Sushi.Mediakiwi/AppCentre/UI/Notification.cs

```diff
@@ -21,17 +18,18 @@ public class Notification : BaseImplementation
         /// </summary>
         public Notification()
         {
-            FilterSelection = 1;
-            //ShowInFullWidthMode = true;
+            FilterSelection = 1;            
 
             ListSearch += Notification_ListSearch;
             ListLoad += Notification_ListLoad;
             ListAction += Notification_ListAction;
         }
 
+        private readonly Mediakiwi.Data.Repositories.Sql.NotificationRepository _repository = new Mediakiwi.Data.Repositories.Sql.NotificationRepository();
```

Comment:
Shouldn't this repository be fetched from DI ?

Reply:
that would break backwards compatability (it would require setting up the DI when updating to this new MK version)

Reply:
I see, but you could use this line as fallback when no INotificationRepository has been added to DI, that way the list can function for as well Elastic and SQL notifications right ?